app/views/users/choose_identity.html.erb

```diff
@@ -7,7 +7,7 @@
 	<%- if @current_user.is_a?(PubcookieUser) -%>
 		<li><%= link_to "Dream Project mentor", update_identity_path(:identity => "Mentor"), :method => :post %></li>
 	<%- else -%>
-		<li><%= link_to "high-school student", update_identity_path(:identity => "Student"), :method => :post %></li>
+		<li><%= link_to "high-school student", update_identity_path(:identity => "Participant"), :method => :post %></li>
```

Comment:
So with the login link functionality, mentees shouldn't ever be using this link, so there should never be any `Student` records created.

For security reasons, we can't let mentees create their own person records. Otherwise, an attacker could log in and act like they are a mentee.

So the workflow will have to be: HSLs or Room Leads collect names and email addresses. They use those to create new `Participants` like we've normally done in the past, there's just less data to enter. They can then send login links to mentees. The mentees will log in, and their account will be associated with the appropriate `Participant` record.